Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Block Pool RADOS Namespace CRD
+title: CephBlockPoolRados Namespace CRD
```

Comment:
> @galexrt How do we increase the width of the TOC on the left column of the docs? With this change, the last three letters of CephBlockPoolRadosNamespace is cut off in that pane.

The space between `CephBlockPoolRados` and `Namespace` is so the table of contents will wrap instead of be cut off. But if we can increase the width of the TOC that would be preferred.

Reply:
I don't think there is an "out of the box" way to do that with the mkdocs material theme itself.